packages/react/slider/src/Slider.tsx

```diff
@@ -663,10 +663,20 @@ function getNextSortedValues(prevValues: number[] = [], nextValue: number, atInd
   return nextValues.sort((a, b) => a - b);
 }
 
-function convertValueToPercentage(value: number, min: number, max: number) {
-  const maxSteps = max - min;
-  const percentPerStep = 100 / maxSteps;
-  return percentPerStep * (value - min);
+/**
+ * Return the `value` position between `min` and `max` as a percentage.
+ * Percentage is clamped between `0` and `100`
+ *
+ * @example
+ * // returns 50
+ * getPositionPercentage(2, 0, 4)
+ *
+ * @example
+ * // returns 100
+ * getPositionPercentage(2000, 0, 4)
+ */
+function getPositionPercentage(value: number, min: number, max: number) {
+  return clamp(100 * ((value - min) / (max - min)), [0, 100]);
```

Comment:
```suggestion
function convertValueToPercentage(value: number, min: number, max: number) {
  const maxSteps = max - min;
  const percentPerStep = 100 / maxSteps;
  const percentage = percentPerStep * (value - min);
  return clamp(percentage, [0, 100]);
```

I'd rather we keep the original name and update the code like this ☝️ as it's clearer IMO.

Also don't think we need the docs above as the code is then self describing.

Reply:
I applied your suggestions as is 👍 
The change in the implementation was unnecessary and readability is subjective.

I do think the jsdoc helped clarify the behaviour of the function: it's not obvious from the name `convertValueToPercentage` that the output will be clamped (130% is a valid percentage after all ^^)

Reply:
Alright np